tests/playwright/enums/widgets.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Questions for clarification: 

What's the advantage of this method?

What is the meaning of 'enums'?